test/widgets/trip_page_builder_test.dart

```diff
@@ -0,0 +1,279 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/drives/pages/drives_page.dart';
+import 'package:motis_mitfahr_app/rides/pages/rides_page.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:motis_mitfahr_app/util/trip/drive_card.dart';
+import 'package:motis_mitfahr_app/util/trip/ride_card.dart';
+
+import '../util/factories/drive_factory.dart';
+import '../util/factories/model_factory.dart';
+import '../util/factories/profile_factory.dart';
+import '../util/factories/ride_factory.dart';
+import '../util/mock_server.dart';
+import '../util/pump_material.dart';
+import '../util/request_processor.dart';
+import '../util/request_processor.mocks.dart';
+
+void main() {
+  late List<Map<String, dynamic>> drives;
+  late Profile profile;
+  final MockRequestProcessor processor = MockRequestProcessor();
+
+  setUpAll(() async {
+    MockServer.setProcessor(processor);
+  });
+  setUp(() async {
+    reset(processor);
+    profile = ProfileFactory().generateFake(id: 1);
+    SupabaseManager.setCurrentProfile(profile);
+  });
+
+  // since the code for the TripPageBuilder is the same for both pages,
+  // we only test the DrivesPage and assume that the RidesPage works the same.
+  // This test is to show that if the ridesPage is called there are rideCards shown.
+  testWidgets('shows Ride card for RidesPage', (WidgetTester tester) async {
+    final List<Map<String, dynamic>> rides = [
+      RideFactory()
+          .generateFake(
+            id: 1,
+            endTime: DateTime.now().add(const Duration(days: 1)),
+            hideInListView: false,
```

Comment:
I mean, that's the default? Why even set it?

Reply:
to make it better to read.